Cargo.toml

```diff
@@ -102,15 +102,14 @@ interprocess = { version = "2", default-features = false }
 indoc = "2"
 lazycell = "1"
 lazy_static = "1.4"
-notify = "6"
+notify = { version = "6", default-features = false, features = ["macos_kqueue"] }
```

Comment:
This is due to a known issue with notify: https://github.com/notify-rs/notify/issues/380, we basically turn off crossbeam channels and switch to std ones instead